src/clj/rems/db/applications.clj

```diff
@@ -667,6 +667,10 @@
   []
   (keys (methods apply-event)))
 
+(defmethod apply-event "save"
+  [application _event]
+  application)
```

Comment:
Why is the event omitted? Bit confusing imho when the function name is apply-event.